apps/prairielearn/src/pages/instructorRequestCourse/instructorRequestCourse.html.ts

```diff
@@ -224,8 +224,24 @@ function CourseNewRequestCard({ csrfToken }: { csrfToken: string }): HtmlValue {
             <label class="form-label" for="cr-ghuser">GitHub Username (optional)</label>
             <input type="text" class="form-control" name="cr-ghuser" id="cr-ghuser" />
             <small class="form-text text-muted">
-              Providing your GitHub username will allow you to edit course content offline. You do
-              not need to provide this if you would like to use the online web editor.
+              Providing your GitHub username will allow you to edit course content offline or run a
+              local installation of PrairieLearn. You do not need to provide this if you would like
+              to use the online web editor. You are encouraged to provide it if you are planning
+              complex questions such as questions using
```

Comment:
Nit:

```suggestion
              complex questions such as those using
```

Reply:
Done in 63276424f848ce5a155a68fc3b4732390e7609c9.